src/Monolog/Clock.php

```diff
@@ -0,0 +1,39 @@
+<?php declare(strict_types=1);
+
+namespace Monolog;
+
+use Psr\Clock\ClockInterface;
+use Monolog\JsonSerializableDateTimeImmutable;
+
+class Clock implements ClockInterface
```

Comment:
I suggest using a less specific name than `Clock` to avoid having it appearing high in IDE completion in projects using Monolog (while Monolog is not meant to become a library providing a clock for generic purposes that would involve making it more customizable)